dockers/README.md

```diff
@@ -0,0 +1,91 @@
+# How to locally test
```

Comment:
I think if there is a script for this local deployment --- that would be awesome.
We can make some assumptions here, e.g., docker is installed, the directory structure is in a specific format, etc.